# HG changeset patch
# User Rik <rik@octave.org>
# Date 1348986518 25200
#      Sat Sep 29 23:28:38 2012 -0700
# Node ID dd086145f4a663f3798beb7c103aa197572d3753
# Parent  41467bebc11444f16644d944894a33b48d329108
build: Remove unusued BUILT_SOURCE_EXTRA variable.

* libinterp/Makefile.am: Remove unusued BUILT_SOURCE_EXTRA variable.

* src/Makefile.am: Remove unusued BUILT_SOURCE_EXTRA variable.

diff --git a/libinterp/Makefile.am b/libinterp/Makefile.am
--- a/libinterp/Makefile.am
+++ b/libinterp/Makefile.am
@@ -55,17 +55,16 @@ BUILT_SOURCES = \
   interpfcn/graphics.h \
   builtin-defun-decls.h \
   operators/ops.cc \
   parse-tree/lex.cc \
   parse-tree/oct-gperf.h \
   parse-tree/oct-parse.cc \
   oct-conf.h \
   version.h \
-  $(BUILT_SOURCES_EXTRA) \
   builtins.cc
 
 BUILT_DISTFILES = \
   parse-tree/oct-gperf.h \
   parse-tree/oct-parse.h
 
 ## Files that are created during build process and installed,
 ## BUT not distributed in tarball.
diff --git a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -50,19 +50,16 @@ mkoctfile_LDADD = $(top_builddir)/libgnu
 
 octave_config_SOURCES =
 nodist_octave_config_SOURCES = octave-config.cc
 octave_config_LDADD = \
   $(top_builddir)/libinterp/corefcn/libcorefcn.la \
   $(top_builddir)/libgnu/libgnu.la \
   $(LIBS)
 
-BUILT_SOURCES_EXTRA = \
-  mkoctfile.cc \
-  octave-config.cc
 else
 bin_PROGRAMS = \
   $(OCTAVE_BINARIES)
 
 bin_SCRIPTS = \
   mkoctfile \
   octave-config
 endif
